### PR TITLE
Removed SQL Request classes

### PR DESCRIPTION
**Pull request by bogensberger**
- removed SQLRequest classes
- moved SQLResponses to test package and made them simple wrappers
- simplified SQLTransportExecutor

### CODE REVIEW

**Comment by bogensberger** [comment]
Please Review @seut @dobe @mfussenegger or @matriv

File: sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java [hunk 1]
```diff
@@ -457,4 +448,30 @@ protected String getIndexSettings(String index) throws IOException {
         return builder.string();
     }
 
+    /**
+     * creates an {@link SQLOperations.Session} on a specific node.
```
Comment (matriv) [inline sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java]: Start with capital letter

File: sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java [hunk 2]
```diff
@@ -457,4 +448,30 @@ protected String getIndexSettings(String index) throws IOException {
...
+     * @param nodeName The name of the node to create the session
+     * @return The created session
+     */
+    SQLOperations.Session createSessionOnNode(String nodeName) {
+        SQLOperations sqlOperations = internalCluster().getInstance(SQLOperations.class, nodeName);
+        return sqlOperations.createSession(null, Option.NONE, DEFAULT_SOFT_LIMIT);
+    }
+
+    /**
+     * creates a {@link SQLOperations.Session} with the given default schema
```
Comment (matriv) [inline sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java]: same here

File: sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java [hunk 3]
```diff
@@ -457,4 +448,30 @@ protected String getIndexSettings(String index) throws IOException {
...
+     * an options list. This is useful if you require a session which differs
```
Comment (matriv) [inline sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java]: and

**Pull request merged**